source/server/options_impl.cc

```diff
@@ -93,14 +99,16 @@ OptionsImpl::OptionsImpl(int argc, char** argv, const HotRestartVersionCb& hot_r
     exit(1);
   }
 
-  if (max_stat_name_len.getValue() < 127) {
-    std::cerr << "error: the 'max-stat-name-len' value specified (" << max_stat_name_len.getValue()
-              << ") is less than the minimum value of 127" << std::endl;
+  if (max_obj_name_len.getValue() < 60) {
+    std::cerr << "error: the 'max-obj-name-len' value specified (" << max_obj_name_len.getValue()
+              << ") is less than the minimum value of 60" << std::endl;
     exit(1);
   }
 
   if (hot_restart_version_option.getValue()) {
-    std::cerr << hot_restart_version_cb(max_stats.getValue(), max_stat_name_len.getValue());
+    std::cerr << hot_restart_version_cb(max_stats.getValue(),
+                                        max_obj_name_len.getValue() +
```

Comment:
It would be nicer if we could call RawStatData::configure(), then use RawStatData::maxNameLength() here, instead of doing the addition.  But I think it's probably more effort than it's worth, because the options_impl doesn't have it's values yet at this point in the function, so probably fine to leave as is, and the tests should catch if this gets messed up somehow.